introduction.md

```diff
@@ -1,2 +1,4 @@
 introduction 
-JavaScript
\ No newline at end of file
+* Git
+* JavaScript
+* PHP
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?